lib/project_types/extension/tasks/get_product.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require "shopify_cli"
+
+module Extension
+  module Tasks
+    class GetProduct < ShopifyCli::Task
+      API_VERSION = "2021-07"
+      GRAPHQL_FILE = "get_variant_id"
+
+      def call(context, shop)
+        response = ShopifyCli::AdminAPI.query(
+          context,
+          GRAPHQL_FILE,
+          shop: shop,
+          api_version: API_VERSION
+        )
```

Comment:
also, I stumbled upon this recent change to AdminAPI: https://github.com/Shopify/shopify-cli/commit/af55aae7d5e4103e7032d6d9f64538966246f625

it seems that `AdminAPI.query` no longer expects the `api_version`?